xc/xc7/tests/bufgmux/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Arty
+
+add_file_target(FILE bufgmux.v SCANNER_TYPE verilog)
+
+add_file_target(FILE arty.pcf)
+add_file_target(FILE arty.sdc)
+
+add_fpga_target(
+  NAME bufgmux_arty
+  BOARD arty-full
+  SOURCES bufgmux.v
+  INPUT_IO_FILE arty.pcf
```

Comment:
Please convert both `.pcf` + `.sdc` to a single `.xdc` file.

Reply:
fixed